Source/SwiftLintFramework/Rules/Style/SortedImportsRuleExamples.swift

```diff
@@ -0,0 +1,96 @@
+internal struct SortedImportsRuleExamples {
+    private static let groupByAttributesConfiguration = ["grouping": "attributes"]
+
+    static let nonTriggeringExamples = [
+        Example("import AAA\nimport BBB\nimport CCC\nimport DDD"),
+        Example("import Alamofire\nimport API"),
+        Example("import labc\nimport Ldef"),
+        Example("import BBB\n// comment\nimport AAA\nimport CCC"),
+        Example("@testable import AAA\nimport   CCC"),
+        Example("import AAA\n@testable import   CCC"),
+        Example("""
+        import EEE.A
+        import FFF.B
+        #if os(Linux)
+        import DDD.A
+        import EEE.B
+        #else
+        import CCC
+        import DDD.B
+        #endif
+        import AAA
+        import BBB
+        """),
+        Example("@testable import AAA\n  @testable import BBB", configuration: groupByAttributesConfiguration),
+        Example("@testable import BBB\n  import AAA", configuration: groupByAttributesConfiguration),
+        Example("@_exported import BBB\n  @testable import AAA", configuration: groupByAttributesConfiguration)
```

Comment:
Examples with non-default configurations should be excluded from the rule's documentation as the configuration isn't shown in the rendered example on the website. There is the flag `excludeFromDocumentation` for this case.